net/nfs-kernel-server/patches/200-uclibc-fix.patch

```diff
@@ -0,0 +1,29 @@
+diff -u --recursive nfs-utils-1.3.3-vanilla/support/export/hostname.c nfs-utils-1.3.3/support/export/hostname.c
+--- nfs-utils-1.3.3-vanilla/support/export/hostname.c	2016-03-31 20:14:11.101902452 -0400
++++ nfs-utils-1.3.3/support/export/hostname.c	2016-03-31 20:15:40.619134622 -0400
+@@ -101,6 +101,7 @@
+ 		.ai_protocol	= (int)IPPROTO_UDP,
+ 		.ai_flags	= AI_NUMERICHOST,
+ 		.ai_family	= AF_UNSPEC,
++		.ai_socktype    = 0,
```

Comment:
It is not needed but does not harm either. When using named struct initializers, then fields not explicitely mentioned are guaranteed to be initialized with zero.